.circleci/config.yml

```diff
@@ -496,7 +497,6 @@ jobs:
       - run:
           name: Run type checkers
           command: |
-            make tstypecheck
             scripts/mypy --report
```

Comment:
nit: Would be useful to have a comment here noting that `tsc` is run by `pre-commit`

Reply:
Changed 👍